packages/frontend/src/context/User.ts

```diff
@@ -92,6 +92,8 @@ export class User {
     save() {
         if (this.id) {
             window.localStorage.setItem('identity', this.id.serializeIdentity())
+        } else {
+            window.localStorage.removeItem('identity')
```

Comment:
We also need to wipe the indexed DB storage.

Reply:
So should I use `db.close()`?

Reply:
ah `db.close()` doesn't change the database, it just closes the handle. You need to use `db.delete('GSTLeaf', { where: {} })` for each collection